mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -87,20 +93,50 @@ class ExperimentView extends Component {
     const paramKeyList = paramKeyFilter.apply(this.props.paramKeyList);
     const metricKeyList = metricKeyFilter.apply(this.props.metricKeyList);
 
-    const columns = ExperimentView.getColumnHeaders(paramKeyList, metricKeyList);
+    const sort = this.state.sort;
+    const columns = ExperimentView.getColumnHeaders(
+      paramKeyList,
+      metricKeyList,
+      this.onCheckAll.bind(this),
+      this.isAllChecked(),
+      this.onSortBy.bind(this),
+      sort);
+
     const metricRanges = ExperimentView.computeMetricRanges(metricsList);
-    const rows = [...Array(runInfos.length).keys()].map((idx) => ({
+    const rows = [...Array(runInfos.length).keys()].map((idx) => {
+      const paramsMap = ExperimentView.toParamsMap(paramsList[idx]);
+      const metricsMap = ExperimentView.toMetricsMap(metricsList[idx]);
+      return {
         key: runInfos[idx].run_uuid,
+        sortValue:
+          (sort.isMetric ? metricsMap :
+          sort.isParam ? paramsMap :
+          runInfos[idx])[sort.key],
         contents: ExperimentView.runInfoToRow(
           runInfos[idx],
           this.onCheckbox,
           paramKeyList,
           metricKeyList,
-          paramsList[idx],
-          metricsList[idx],
-          metricRanges)
-      })
-    );
+          paramsMap,
+          metricsMap,
+          metricRanges,
+          !!this.state.runsSelected[runInfos[idx].run_uuid])
+      }
+    });
+    rows.sort((a, b) => {
+      let x = a.sortValue;
+      let y = b.sortValue;
+      if(x === undefined)
+        return 1;
+      if(y === undefined)
+        return -1;
+      if(!this.state.sort.ascending) {
```

Comment:
Nit: try to follow the coding style for the other code here, which is to leave a space after `if`